packages/web/src/components/ui/PackagePreview/PackagePreview.tsx

```diff
@@ -82,27 +82,29 @@ export default function PackagePreview({
       >
         <div className={styles.content}>
           <div className={styles.contentInner}>
-            <div className={styles.stat}>
-              <div className={styles.statHeader}>
-                <Icon kind='script' color='#8E8AA0' className={styles.statIcon} />
-                Script
+            {(pkg.containingScripts?.length ?? 0) > 0 && (
+              <div className={styles.stat}>
+                <div className={styles.statHeader}>
+                  <Icon kind='script' color='#8E8AA0' className={styles.statIcon} />
+                  Script
+                </div>
+                {detailsLoading ? (
+                  <ScriptSkeleton />
```

Comment:
Also, seems like `<ScriptSkeleton>` will never be shown with current logic and data flow

Reply:
True. I suggest dealing with it a bit later though.